src/audio/dcblock/dcblock.c

```diff
@@ -183,17 +183,6 @@ static int dcblock_process(struct processing_module *mod,
 	return 0;
 }
 
-/* init and calculate the aligned setting for available frames and free frames retrieve*/
-static inline void dcblock_set_frame_alignment(struct audio_stream *source,
-					       struct audio_stream *sink)
-{
-	const uint32_t byte_align = 1;
-	const uint32_t frame_align_req = 1;
-
-	audio_stream_init_alignment_constants(byte_align, frame_align_req, source);
-	audio_stream_init_alignment_constants(byte_align, frame_align_req, sink);
```

Comment:
if this part can be removed, how about others? like eq_fir or eq_iir?

Reply:
I think the purpose of this PR is remove all the default alignment set(all byte_align = 1 && frame_align_req = 1 ) and set the default value while buffer is allocated.

Reply:
The eq_fir/iir usage passes a "2" for frame alignment, with a comment indicating that it wants to process pairs of frames (see e.g. https://github.com/thesofproject/sof/blob/main/src/audio/eq_fir/eq_fir.c#L399).  That seems like a legitimate usage to me (not knowing anything else about the algorithm).  The goal with this patch is just to remove the requirement for **all** components to call this even when they don't have special alignment requirements.